include/utilities.h

```diff
@@ -79,6 +79,27 @@ bool copy_file(const std::string &files, const std::string &destination);
  */
 bool rename_file(const std::string &file, const std::string &new_file);
 
+/**
+ * This class rewrite @p n_max lines of output
+ */
+class fixed_lines
+{
+public:
+  fixed_lines(int, std::ostream &stream_out = std::cout);
+  int get_current_line();
+  void goto_previous_line(int n_line = 1, bool erase=false);
+  void print_line(std::string &txt, bool erase=true);
+
```

Comment:
I would like to overload the operator << but I got a lot of troubles doing it...